AirCasting/Services/UpdateSessionParams.swift

```diff
@@ -0,0 +1,70 @@
+//
+//  UpdateSessionParams.swift
+//  AirCasting
+//
+//  Created by Lunar on 01/04/2021.
+//
+
+import Foundation
+import CoreData
+
+class UpdateSessionParamsService {
+    
+    var context: NSManagedObjectContext {
+        PersistenceController.shared.container.viewContext
+    }
+    
+    func updateSessionsParams(session: Session, output: FixedSession.FixedMeasurementOutput) {
+        let dateFormatter = ISO8601DateFormatter.defaultLong
+    
+        session.id = Int64(output.id)
```

Comment:
Do we need to save this id from web to our db? I think session id is set automatically and we shouldn't change it. It doesn't have to be the same in our db as it is in web.

Reply:
Hmm, I guess not.